File: old/Team/1.java
import java.util.Scanner;
class team{
	public static int chk(int[] ans,int k){
		int i,n=0;
		for(i=0;i<3;i++){
			if(ans[i]==1){
				n++;
			}
		}
		return n;
	}
	public static void main(String[] arg){
		Scanner in = new Scanner(System.in);
		int i,n,problem=0;
		int[] ans = new int[100];
		n = in.nextInt();
		for(i=0;i<n;i++){
			ans[0] = in.nextInt();
			ans[1] = in.nextInt();
			ans[2] = in.nextInt();
			if(chk(ans,n)>1){
				problem++;
			}
		}
		System.out.print(problem);
	}
}

File: old/Team/1.cpp
#include <stdio.h>
int chk(int ans[100],int k){
	int i,n=0;
	for(i=0;i<3;i++){
		if (ans[i]==1)
			n++;
		}
		return n;
	}
int main(){
	int n,problem=0,i,ans[100];
	scanf("%d",&n);
	for (i=0;i<n;i++){
		scanf("%d%d%d",&ans[0],&ans[1],&ans[2]);
		if(chk(ans,n)>1){
			problem++;
		}
	}
	printf("%d",problem);
}